lib/core/src/control/event_loop.rs

```diff
@@ -40,18 +47,18 @@ impl EventLoop {
     /// Init the event loop.
     fn init(self) -> Self {
         let data = Rc::downgrade(&self.rc);
-        let main = move |_| { data.upgrade().map(|t| t.borrow_mut().run()); };
+        let main = move |time_ms| { data.upgrade().map(|t| t.borrow_mut().run(time_ms)); };
         with(self.rc.borrow_mut(), |mut data| {
             data.main = Some(Closure::new(main));
-            data.run();
+            data.run(0.0);
```

Comment:
This 0.0 may be really surprising. Either pass here "performance.now" value, or just register main and let wait for the first frame.